web-console/src/routes/(authenticated)/(pipelines)/create/+page.ts

```diff
@@ -1,31 +1,33 @@
 import { goto } from '$app/navigation'
 import { base } from '$app/paths'
 import { useTryPipeline } from '$lib/compositions/pipelines/useTryPipeline'
-import type { PipelineDescr } from '$lib/services/manager'
+import type { Demo } from '$lib/services/manager'
 import { getDemos } from '$lib/services/pipelineManager'
 
 export async function load({ url, parent }) {
   await parent()
-  const pipeline: PipelineDescr | undefined = await (() => {
+  const pipeline: Omit<Demo, 'title'> | undefined = await (() => {
     const name = url.searchParams.get('name')
     if (!name) {
       return undefined
     }
-    const code = url.searchParams.get('code')
-    if (code) {
+    const program_code = url.searchParams.get('program.sql') ?? url.searchParams.get('code')
```

Comment:
Rather than adding a new naming scheme, why not use the direct equivalents used by the API: `?program_code=...&udf_rust=...&udf_toml=...`

Reply:
Well it's a semantic difference between specifying api object fields vs specifying file contents, and I decided to go with the latter